frontend/src/core/models/task/task-create-view-model.ts

```diff
@@ -0,0 +1,7 @@
+export interface TaskCreateViewModel {
+    currentUser?: number;
+    project?: [string, string, number];
+    issueType?: [string, string, number];
```

Comment:
Why not enum ?

Reply:
It's not not my code too. If i change it, there will be need to fix another page.